### PR TITLE
generalized api request

### PR DESCRIPTION
**Pull request by rahimrasool**

### CODE REVIEW

File: code/RouteWithTime.py [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+
+    # Take file name from terminal
+    df_list = read_csv_files_to_dataframes()
+
+    return df_list, mapbox_token
+
+
+def request_time(df, mapbox_token):
+    col_duration = [300]
+    total_time = 300
```
Comment (rahimrasool) [inline code/RouteWithTime.py:73]: What does 300 here mean?

Comment (genieugod) [reply code/RouteWithTime.py:73]: For the first stop, add a loading time of 5min (300 seconds) I will add a comment here.

**Comment by rahimrasool** [comment]
Kindly resave the conflicts

**Comment by rahimrasool** [comment]
Please resolve the conflicts

**Pull request closed without merging**